Resources/Prototypes/Entities/Mobs/base.yml

```diff
@@ -60,6 +60,16 @@
         damage: 400
       behaviors:
       - !type:GibBehavior { }
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Heat
+        damage: 1500
+      behaviors:
+      - !type:SpawnEntitiesBehavior
```

Comment:
wont this not delete the mob

Reply:
All this currently does is spawn an ash entity when the mob reaches the threshhold

Reply:
ah my bad